test/unit/create-app-definition-api-test.js

```diff
@@ -125,6 +125,27 @@ describe('createAppDefinitionApi', () => {
     })
   })
 
+  test('API call createAppBundle with delivery functions', async () => {
+    const { api, entitiesMock } = setup(Promise.resolve({}))
+    const appBundleWithDeliveryFns = {
+      ...appBundleMock,
+      deliveryFunctions: [
+        {
+          id: 'myCustomId',
+          name: 'My Action',
+          description: 'Action uploaded with bundle',
```

Comment:
```suggestion
          name: 'My Delivery function',
          description: 'Delivery function uploaded with bundle',
```

For sure not a blocker 😅